platform-sdk/swirlds-common/src/main/java/com/swirlds/common/crypto/Cryptography.java

```diff
@@ -26,6 +32,19 @@ public interface Cryptography {
     /** The default digest type */
     DigestType DEFAULT_DIGEST_TYPE = DigestType.SHA_384;
 
+    /**
+     * Creates a new {@link Cryptography} instance using the given configuration.
+     *
+     * @param configuration
+     * 		the configuration to use
+     * @return a new {@link Cryptography} instance
+     */
+    @NonNull
+    static Cryptography create(@NonNull final Configuration configuration) {
```

Comment:
Can we create a CryptographyFactory class for that method and directly annotate it with deprecated?

Reply:
Done.